src/components/homepage/commitments-entry.tsx

```diff
@@ -0,0 +1,132 @@
+import React from "react";
+import { css } from "@emotion/core";
+import { mq, BreakPoint, screenSizes } from "../../util/mq";
+import useWindowSize from "../../util/useWindowSize";
+
+const commitmentsEntryStyles = css`
+  margin-top: 5.25rem;
+  font-size: 1.125rem;
+  line-height: 1.875rem;
+  text-align: left;
+
+  > div {
+    column-gap: 1rem;
+    display: grid;
+  }
+
+  .right-shift {
+    grid-template-columns: 6fr 5fr;
+
+    .homepage-image-wrapper,
+    svg {
+      justify-self: end;
+    }
+  }
+
+  .left-shift {
+    grid-template-columns: 5fr 6fr;
+
+    .homepage-image-wrapper,
+    svg {
+      justify-self: start;
+    }
+  }
```

Comment:
I think there's a much better way to do this using `grid-template-area` so we can set up the 11 columns divided into 4 areas in the `5, 6 / 6, 5` pattern. I plan to push up changes to this branch to show this.

Reply:
Will do this in a later PR if I get around to it.